corehq/apps/data_cleaning/templates/data_cleaning/tables/table_with_controls.html

```diff
@@ -57,6 +57,12 @@
           class="btn btn-outline-danger"
           type="button"
           x-show="numRecordsSelected > 0"
+          hx-post="{{ request.path_info }}{% querystring %}"
+          hq-hx-action="deselect_all"
+          hx-target="{% if table.container_id %}#{{ table.container_id }}{% else %}div.table-container{% endif %}"
```

Comment:
Total nitpick, but I'm surprised to see `div.table-container` instead of `.table-container`, do you have any thoughts on preferring one or the other?

Reply:
i just copied what I had elsewhere already, but i will note this and update later. i'm surprised i also opted for `div.table-container` instead of `.table-container` but maybe this was a lingering artifact from the prototype that i failed to scrutinize. noting this comment for future cleanup (as i want to make sure it's consistent everywhere)